app/models/movie.rb

```diff
@@ -1,2 +1,5 @@
 class Movie < ApplicationRecord
+  has_and_belongs_to_many :actors
+  has_many :posters, dependent: :destroy
+  accepts_nested_attributes_for :posters, allow_destroy: true
```

Comment:
add an empty line above this.
Never shy away from using spaces(line breaks) to add structure and neatness to your code.